rpc/send_transaction_test.go

```diff
@@ -68,7 +69,7 @@ func TestRPC_SendIntent_SendTransaction(t *testing.T) {
 
 	c := proto.NewWaasAuthenticatorClient(srv.URL, http.DefaultClient)
 	header := make(http.Header)
-	header.Set("X-Access-Key", newRandAccessKey(tenant.ProjectID))
+	header.Set("X-Sequence-Project", strconv.Itoa(int(tenant.ProjectID)))
```

Comment:
hmm, what is X-Sequence-Project....? I dont remember, is that a thing..?

Reply:
the ingress proxy parses the access key (as it uses quotacontrol) and passes it to waas-authenticator in the `X-Sequence-Project` header - this is already in place